.semaphore/semaphore.yml

```
@@ -5,157 +5,93 @@ agent:
     type: e1-standard-2
     os_image: ubuntu1804
 blocks:
-  - name: Run other linters
-    dependencies: []
+  - name: Run linters and tests
     task:
-      env_vars:
-        - name: LINTJE_VERSION
-          value: "0.3.0"
+      prologue:
+        commands:
+          - checkout
       jobs:
         - name: Git Lint (Lintje)
+          env_vars:
+            - name: LINTJE_VERSION
+              value: "0.3.0"
           commands:
-            - checkout
             - script/install_lintje
             - $HOME/bin/lintje $SEMAPHORE_GIT_COMMIT_RANGE
-  - name: Run Elixir linters
-    dependencies: []
-    task:
-      prologue:
-        commands:
-          - sem-version erlang 23.2
-          - sem-version elixir 1.11.3
-          - elixir -v
-          - checkout
-          - mix local.rebar --force
-          - mix local.hex --force
-          - mix deps.get
-
-      jobs:
+        - name: mix compile --warnings-as-errors
+          commands:
+            - ERLANG_VERSION=24.0 ELIXIR_VERSION=1.12.2 . bin/setup
```

```suggestion
            - ERLANG_VERSION=24.0 ELIXIR_VERSION=1.12.2 bin/setup
```

What's the dot for? I think we can remove it in all jobs

---

I got it from the Elixir library

---

it's probably not needed there either. Can you try without and if it works update the elixir package as well?

---

Changes done.